account/manager.go

```diff
@@ -2084,3 +2163,24 @@ func makeTxnLabel(labelPrefix, contextLabel string) string {
 
 	return label
 }
+
+// serializeWitness turns a wire witness into its serialized form.
+func serializeWitness(witness wire.TxWitness) ([]byte, error) {
+	var buf bytes.Buffer
+	if err := psbt.WriteTxWitness(&buf, witness); err != nil {
```

Comment:
Ah cool I was actually looking for something like this in a related project (along w/ reading an outpoint serialized in bitcoin wire format), nice to know it was in the `psbt` package.